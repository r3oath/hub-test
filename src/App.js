// @flow

import * as React from 'react';
import Button from './components/Button';

export default function App() {
  return (
    <div>
      <h1>Hello World!</h1>
      <Button>Get started</Button>
    </div>
  );
}
