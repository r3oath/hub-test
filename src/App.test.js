import * as React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('works without crashing', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
