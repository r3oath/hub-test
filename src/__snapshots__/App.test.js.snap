// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works without crashing 1`] = `
<h1>
  Hello World!
</h1>
`;
